File: website-backend/src/main/java/marshall/project/model/TradeItem.java
package marshall.project.model;

public class TradeItem {
	private Integer itemId;
	private String name;
	private String description;
	private String tradeUrl;
	private String priceHistoryUrl;
	private Integer userId;
	private String collectionName;
	private String leagueName;

	public Integer getUserId() {
		return userId;
	}
	public void setUserId(Integer userId) {
		this.userId = userId;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public String getTradeUrl() {
		return tradeUrl;
	}
	public void setTradeUrl(String tradeUrl) {
		this.tradeUrl = tradeUrl;
	}
	public String getPriceHistoryUrl() {
		return priceHistoryUrl;
	}
	public void setPriceHistoryUrl(String priceHistoryUrl) {
		this.priceHistoryUrl = priceHistoryUrl;
	}
	public Integer getItemId() {
		return itemId;
	}
	public void setItemId(Integer itemId) {
		this.itemId = itemId;
	}
	public String getCollectionName() {
		return collectionName;
	}
	public void setCollectionName(String collectionName) {
		this.collectionName = collectionName;
	}
	public String getLeagueName() {
		return leagueName;
	}
	public void setLeagueName(String leagueName) {
		this.leagueName = leagueName;
	}
}

File: website-frontend/my-app/src/app/poe-trade/TradeItem.ts
export class TradeItem {
    itemId:number;
    name:string;
    description:string;
    tradeUrl:string;
    priceHistoryUrl:string;
    userId:number;
    collectionName:String;
    leagueName:String;

    constructor(itemId:number, name:string, description:string, tradeUrl:string, priceHistoryUrl:string, userId:number, collectionName:String, leagueName:String) {
        this.itemId = itemId;
        this.name = name;
        this.description = description;
        this.tradeUrl = tradeUrl;
        this.priceHistoryUrl = priceHistoryUrl;
        this.userId = userId;
        this.collectionName = collectionName;
        this.leagueName = leagueName;
    }

}